web/src/main/java/com/epam/esm/CertificateController.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.esm;
+
+import com.epam.esm.dto.CertificateDTO;
+import com.epam.esm.exceptionHandling.NoSuchResourceException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/certificates")
+public class CertificateController {
+
+    @Autowired
+    private CertificateService certificateService;
+
+    @GetMapping
+    public List<CertificateDTO> showAllCertificate() {
+        return certificateService.getAllCertificates();
+    }
+
+    @PostMapping("/partOfCertificate")
+    public List<CertificateDTO> showCertificateByPartOfDescription(@RequestBody CertificateDTO certificateDTO) {
```

Comment:
What is the reason to pass full certificateDto if you only need a name and description? Also this is a read operation, why did you decide to choose post method?